src-ui/lib/components/BooksTable.svelte

```diff
@@ -56,7 +58,18 @@
   }
 </script>
 
-<svelte:document on:scan={handleScan} use:listenForBarcodes />
+
+<button on:click={() => onScan.simulate(document, '1234567890123')}>
```

Comment:
Should we disable the button if there's a scan simulation in progress?

Reply:
You're probably right about disabling the button - I'll fix that. 
the function (simulate) is already included in the scan library itself.